02/Task02.js

```diff
@@ -1,9 +1,43 @@
 import React from 'react';
+import {Switch, Route ,useParams} from "react-router-dom";
+import products from './../src/products.json';
 
 const Task02 = () => {
-    return (
-        <h1>Task02</h1>
-    );
+
+	const Product = () => {
+		const {id} = useParams();
+		const [productList = null] = products.filter((item) => item.id === parseInt(id));
```

Comment:
To raczej nie jest `productList`,a  `produkt` - bo jest tylko jeden. Nazwa zmiennej wprowadza w błąd ;)